web/react/components/access_history_modal.jsx

```diff
@@ -3,7 +3,8 @@
 
 var UserStore = require('../stores/user_store.jsx');
 var AsyncClient = require('../utils/async_client.jsx');
-var Utils = require('../utils/utils.jsx');
+var LoadingScreen = require('./loading_screen.jsx');
+var utils = require('../utils/utils.jsx');
```

Comment:
I think for this the style guide wants lower case. The convention is that class names (and therefore react component names) are UpperCamelCase. Utils is nether so I think lower case is what it wants. But we can override this in our style guide if we want. (this is unfortunately not enforced by ESLint.)